content/articles/oop-in-java-abstraction-and-encapsulation/index.md

```diff
@@ -1,350 +1,482 @@
-#**The concept of OOP In Java Getting Started with ABSTRACTION and ENCAPSULATION**
+# The concept of OOP In Java Getting Started with ABSTRACTION and ENCAPSULATION
 
-In Java, everything is considered an object and is treated as such; a real-life entity.
+In Java, everything is an object and as such; a real-life entity.
 
-Object-Oriented Programming(OOP) unlike Procedural programming is an approach or a programming paradigm where everything is treated as Classes(Logical construct) and Objects(Real-life entities or instances of a class).
+Object-Oriented Programming (OOP) unlike Procedural programming is an approach. Or a programming paradigm where everything is an Object (Real-life entities). OOP tries to model real-life objects.
+Objects come from class, a form of generalization but the real thing is the Object. What then is a class?
 
-Class is a template from which objects are created. 
-The process of creating Objects from a class is referred to as Instantiation
+Class is a template from which objects are created 
 
-In this tutorial, you will learn about Abstraction and Encapsulation and build more on your already existing knowledge of the concept of OOP, and to say the least, this will be a nice piece to start with for absolute java beginners.
+The process of creating Objects from a class is Instantiation
+
+In this tutorial, you will learn about Abstraction, Encapsulation, Objects, and classes. 
+The concept of OOP, to say the least, will be a nice piece to grab for absolute java beginners.
 
 ## Prerequisites
+
 The reader should have:
-- [At least a basic knowledge of Java]
-- [Have Java installed on your computer]
-- [Have any IDE of your choice]
+
+- At least a basic knowledge of Java
+
+- Have Java installed on your computer
+
+- Have any IDE of your choice
 
 ## Objectives
+
 At the end of this first OOP tutorial series readers should be able to:
-- [Differentiate between classes and objects]
-- [Understand Abstraction and Encapsulation]
-- [Create objects from classes]
-- [Have a working knowledge of what OOP is about]
-- [Create abstract classes and interfaces]
 
+- Differentiate between classes and objects
+
+- Understand Abstraction and Encapsulation
+
+- Create objects from classes
+
+- Have a working knowledge of what OOP is about
+
+- Create abstract classes and interfaces
+
+### CLASS
 
-###**CLASS**
 A class is a template from which an object is created. A class serves as a blueprint for object creation.
```

Comment:
> A class is a template from which an object is created. A class serves as a blueprint for object creation.

I would join these sentences together like so for better flow:

> A class is a template from which we create objects, serving as a blueprint for object creation.

Also, I removed an instance of [passive voice](https://www.hamilton.edu/academics/centers/writing/seven-sins-of-writing/1) for a little more clarity.